seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -1217,7 +1217,7 @@
         "We can also use the <code>+=</code> operator to <dfn>concatenate</dfn> a string onto the end of an existing string variable. This can be very helpful to break a long string over several lines.",
         "<strong>Note</strong><br>Watch out for spaces. Concatenation does not add spaces between concatenated strings, so you'll need to add them yourself.",
         "<hr>",
-        "Build <code>myStr</code> over several lines by concatenating these two strings:<br><code>\"This is the first sentence. \"</code> and <code>\"This is the second sentence.\"</code> using the <code>+=</code> operator."
+        "Build <code>myStr</code> over several lines by concatenating these two strings: <code>\"This is the first sentence. \"</code> and <code>\"This is the second sentence.\"</code> using the <code>+=</code> operator. Use the <code>+=</code> operator similar to how it is shown in the editor. Start by assigning the first string to <code>myStr</code>, then add on the second string."
```

Comment:
@systimotic no worries. I forgot to respond. Again, I would say `then add on` is very informal. A better and more explicit way to explain the task would be:

> Start by assigning the first string to <code>myStr</code>, then add on the second string to the end of the first string using the `+=` operator.

This is a lot wordier but more explicit on what needs to be done. But yes, the `then add on` is sufficient though.